e2e/.gitignore

```
@@ -0,0 +1,5 @@
+node_modules/
+/test-results/
+/playwright-report/
+/blob-report/
+/playwright/.cache/
```

add .env to gitignore?
```suggestion
node_modules/
/test-results/
/playwright-report/
/blob-report/
/playwright/.cache/
*.env
```

---

Sure. Should be covered by the parent gitignore, but always good to be safe.